lab-trent/app/index.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+require('./partials/main.scss');
+
+const angular = require('angular');
+const cowsay = require('cowsay-browser');
+
+const cowsayApp = angular.module('cowsayApp', []);
+
+cowsayApp.controller('CowsayController', [CowsayController]);

```

Comment:
My only critique is that I'd like you to be passing in `$log` to your controller, and using `$log.debug()` statements. The nice thing about them is that they can be ignored in production renderings, so we can have them for development.